udata/core/dataset/csv.py

```diff
@@ -1,4 +1,3 @@
-from udata.core.discussions.models import Discussion
 from udata.frontend import csv
```

Comment:
You could actually import `DiscussionCsvAdapter` here for backward compatibility

Reply:
ruff doesn't like that it is imported but unused 🙃